src/ServiceFabric/ServiceFabric/Commands/AddAzureRmServiceFabricNodeType.cs

```diff
@@ -202,7 +202,7 @@ private void CreateVmss(string clusterId)
                      Tags = GetServiceFabricTags(),
                      UpgradePolicy = new UpgradePolicy()
                      {
-                         Mode = UpgradeMode.Automatic
+                         Mode = Azure.Commands.Common.Compute.Version_2018_04.Models.UpgradeMode.Automatic
```

Comment:
@LukeSlev can you consider renaming the class UpgradeMode to ApplicationUpgradeMode? so we don't have this conflict and others in the future?